bundles/af-core/src/main/java/com/adobe/cq/forms/core/components/internal/models/v2/form/FormContainerImpl.java

```diff
@@ -103,6 +104,7 @@ public class FormContainerImpl extends AbstractContainerImpl implements FormCont
     protected void initFormContainerModel() {
         if (request != null) {
             contextPath = request.getContextPath();
+            request.setAttribute("formContainerPath", this.getPath());
```

Comment:
Can you create a constant for this ? Please prefix the constant with the context, for example, the constant name could be REQ_ATTR_FORMCONTAINER_PATH

Reply:
ok